### ISSUE TITLE
DOMException on CustomElementRegistry when using multiple modules

### ISSUE THREAD

**Issue by 00155837364765867** [vasturiano/react-force-graph#501]
When using react-force-graph in two separate module packages, then those are installed into a parent application, 
An exception is thrown after the first instance is accessed/loaded and then the second is accessed/loaded (the order doesn't matter).
When the second node_module package is loaded, this exception is thrown:
Uncaught DOMException: Failed to execute 'define' on 'CustomElementRegistry': the name "a-node" has already been used with this registry.
In v1.44.3, this occurs in react-force-graph.js on line 26641:
![image](https://github.com/vasturiano/react-force-graph/assets/32715712/5770205a-03b6-4f79-80ec-8500695697c0)
[N.B. The actual execution may be using the minified version, but the code base should match for the same version]

If I refactor 'a-node' in one of the modules to say 'x-node', the exception is then thrown for the next conflicting variable (in my case it is 'a-assets'). This isolates the issue as being within the react-force-graph es code.

**To Reproduce**
Create an exported node_module package that references react-force-graph.
Build, package and publish it.
Then, build a second separate node_module package also referencing react-force-graph.
Build, package and publish that.

Now install both these packages into a parant application that uses these two node_module libs.
Depending how this parent app is written:
Access the parent app such that one of the node_module packages is loaded (e.g. show-me-graph-1).
This will load fine, no problems relating to this issue.

Now access the parent app such that the second node_module package is loaded (e.g. show-me-graph-2).
This will throw the error shown above.

**Expected behavior**
it would be great if the ES code could check the registry to see if the relevant variables and primitives etc. are already registered in the CustomElementRegistry (e.g. a-node, a-assets etc), then skip the registration if already registered.

**Screenshots**
When this error occurs, the screen is blank.

**Desktop (please complete the following information):**
 - OS: All
 - Browser Chrome, Firefox, Edge, Safari
 - Version react-force-graph v1.44.3

**Smartphone (please complete the following information):**
N/A

**Additional context**
In my test environment for this issue, I have set the react-force-graph typically as a peerDependency in package.json, and the in parent app as a dependency. I have tried various combinations of build/install/package options but the same error occurs in the same manner.

npm: 9.5.1
vite: 4.5.2
ract: 18.2.0

**Comment by vasturiano** [comment]
@00155837364765867 this is likely related to multiple modules of [A-Frame](https://aframe.io/) in your dependency tree.

Do you confirm that you have more than one instance of that module installed in your app?

If so, that's probably the source of the issue, due to the side-effects that that particular module has:
https://aframe.io/docs/1.5.0/core/globals.html#aframe_properties_anode

Are you actually using the VR mode in this module? If you're not, you might be better off just importing the module specific to the mode you're using to circumvent this issue. F.e. if you want to just import the 3D mode:
```js
import ForceGraph3D from 'react-force-graph-3d';
```

**Comment by 00155837364765867** [comment]
Hi,
Thanks for your quick response!

_Do you confirm that you have more than one instance of that module installed in your app?_
Well, the behaviour of the exception would suggest there is more than one instance being loaded (but only after a second is referenced at runtime).

_you might be better off just importing the module specific to the mode you're using to circumvent this issue_
Yes, that is indeed what I do:
  import { ForceGraph2D } from 'react-force-graph';

I got it working by removing react-force-graph and using the standalone package react-force-graph-2d.

I had missed this important sentence on your main page: 

_For dependency convenience, all of the components are also available as stand-alone packages: react-force-graph-2d, react-force-graph-3d, react-force-graph-vr and react-force-graph-ar._

Once I did that and cleaned my node_modules, it removed all the aframe stuff.
This certainly skirts 'round my symptom (thank for your pointer!) but I suppose the underlying issue remains, I guess it is what it is using global registries!

Thank you for your help, and more importantly for your excellent module!
